app/store/ephemeral_store.ts

```diff
@@ -10,6 +10,12 @@ class EphemeralStore {
     creatingChannel = false;
     creatingDMorGMTeammates: string[] = [];
 
+    // As of today, the server sends a duplicated event to add the user to the team.
+    // If we do not handle this, this ends up showing some errors in the database, apart
+    // of the extra computation time. We use this to track the events that are being handled
+    // and make sure we only handle one.
+    addingTeam: {[id: string]: boolean} = {};
```

Comment:
you should set a getter and a removal function to access this prop... the way is currently done can be error prone, thus the reason why have getters and setters for the other properties as well.